docs/PIL.rst

```diff
@@ -69,6 +69,15 @@ can be found here.
     :undoc-members:
     :show-inheritance:
 
+:mod:`~PIL.ImageMode` Module
+-----------------------------
+
+.. automodule:: PIL.ImageMode
+    :members:
+    :member-order: bysource
```

Comment:
```suggestion
```
This isn't included on all the automodules, and it doesn't make any difference for ImageMode.